t3/utils/flux.py

```diff
@@ -295,6 +295,15 @@ def set_batch_p(gas: ct.Solution,
     network.rtol = r_tol
     return network, reactor
 
+def closest_bigger_number(array, target):
```

Comment:
please add a docstring and typehints for each new function, e.g., as we have for `run_jsr` below